src/_posts/languages/php/2000-01-01-start.md

```diff
@@ -149,6 +149,7 @@ Scalingo supports the following versions of Composer:
 - 2.3.10
 - 2.4.4
 - 2.5.1
+- 2.5.2
```

Comment:
I would remove the previous version of the 2.5.x branch. It's not that interesting to keep track of it.
```suggestion
- 2.5.2
```